firefox_puppeteer/ui/update_wizard/wizard.py

```diff
@@ -0,0 +1,347 @@
+
+    @property
+    def installed(self):
+        """The installed panel.
+
+        :returns: :class:`InstalledPanel` instance.
+        """
+        return self._create_panel_for_id('installed')
+
+    @property
+    def license(self):
+        """The license panel.
+
+        :returns: :class:`LicensePanel` instance.
+        """
+        return self._create_panel_for_id('license')
+
+    @property
+    def manual_update(self):
+        """The manual update panel.
+
+        :returns: :class:`ManualUpdatePanel` instance.
+        """
+        return self._create_panel_for_id('manualUpdate')
+
+    @property
+    def no_updates_found(self):
+        """The no updates found panel.
+
+        :returns: :class:`NoUpdatesFoundPanel` instance.
+        """
+        return self._create_panel_for_id('noupdatesfound')
+
+    @property
+    def plugin_updates_found(self):
+        """The plugin updates found panel.
+
+        :returns: :class:`PluginUpdatesFoundPanel` instance.
+        """
+        return self._create_panel_for_id('pluginupdatesfound')
+
+    @property
+    def updates_found_basic(self):
+        """The updates found panel.
+
+        :returns: :class:`UpdatesFoundPanel` instance.
+        """
+        return self._create_panel_for_id('updatesfoundbasic')
+
+    @property
+    def updates_found_billboard(self):
+        """The billboard panel shown if an update has been found.
+
+        :returns: :class:`UpdatesFoundBillboardPanel` instance.
+        """
+        return self._create_panel_for_id('updatesfoundbillboard')
+
+    @property
+    def panels(self):
+        """List of all the available :class:`Panel` instances.
+
+        :returns: List of :class:`Panel` instances.
+        """
+        panels = self.marionette.execute_script("""
+          let wizard = arguments[0];
+          let panels = [];
+
+          for (let index = 0; index < wizard.children.length; index++) {
+            if (wizard.children[index].id) {
+              panels.push(wizard.children[index].id);
+            }
+          }
+
+          return panels;
+        """, script_args=[self.element])
+
+        return [self._create_panel_for_id(panel) for panel in panels]
+
+    @property
+    def selected_index(self):
+        """The index of the currently selected panel.
+
+        :return: Index of the selected panel.
+        """
+        return int(self.element.get_attribute('pageIndex'))
+
+    @property
+    def selected_panel(self):
+        """A :class:`Panel` instance of the currently selected panel.
+
+        :returns: :class:`Panel` instance.
+        """
+        return self._create_panel_for_id(self.element.get_attribute('currentpageid'))
+
+
+class Panel(UIBaseLib):
+
+    def __eq__(self, other):
+        return self.element.get_attribute('id') == other.element.get_attribute('id')
+
+    def __ne__(self, other):
+        return self.element.get_attribute('id') != other.element.get_attribute('id')
```

Comment:
We have a couple of "Panel" classes now that are near duplicates, let's consolidate.